<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">QLabel {
    font: 14px &quot;Helvetica Neue&quot;; /* 字体样式 */
    color: #333; /* 字体颜色 */
}</string>
        </property>
        <property name="text">
         <string>IP:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_Port">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 1px solid #C0C0C0; /* 边框颜色 */
    border-radius: 5px; /* 圆角 */
    padding: 5px; /* 内边距 */
    background-color: #FFFFFF; /* 背景颜色 */
}

QLineEdit:focus {
    border: 1px solid #0A84FF; /* 聚焦时边框颜色 */
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">QLabel {
    font: 14px &quot;Helvetica Neue&quot;; /* 字体样式 */
    color: #333; /* 字体颜色 */
}</string>
        </property>
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_IP">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 1px solid #C0C0C0; /* 边框颜色 */
    border-radius: 5px; /* 圆角 */
    padding: 5px; /* 内边距 */
    background-color: #FFFFFF; /* 背景颜色 */
}

QLineEdit:focus {
    border: 1px solid #0A84FF; /* 聚焦时边框颜色 */
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QPushButton" name="pushButton_ConnectToServer">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #0A84FF; /* 按钮颜色 */
    color: white; /* 字体颜色 */
    border: none; /* 移除边框 */
    border-radius: 5px; /* 圆角 */
    padding: 10px 20px; /* 内边距 */
    font: 16px &quot;Helvetica Neue&quot;; /* 字体样式 */
}

QPushButton:hover {
    background-color: #007AFF; /* 悬停时颜色 */
}</string>
        </property>
        <property name="text">
         <string>连接服务器</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_ChooseFile">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 1px solid #C0C0C0; /* 边框颜色 */
    border-radius: 5px; /* 圆角 */
    padding: 5px; /* 内边距 */
    background-color: #FFFFFF; /* 背景颜色 */
}

QLineEdit:focus {
    border: 1px solid #0A84FF; /* 聚焦时边框颜色 */
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_ChooseFile">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #0A84FF; /* 按钮颜色 */
    color: white; /* 字体颜色 */
    border: none; /* 移除边框 */
    border-radius: 5px; /* 圆角 */
    padding: 10px 20px; /* 内边距 */
    font: 16px &quot;Helvetica Neue&quot;; /* 字体样式 */
}

QPushButton:hover {
    background-color: #007AFF; /* 悬停时颜色 */
}</string>
         </property>
         <property name="text">
          <string>选择文件</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar_SentFile">
      <property name="styleSheet">
       <string notr="true">QProgressBar {
    border: none; /* 移除边框 */
    border-radius: 10px; /* 更大的圆角 */
    background-color: #E0E0E0; /* 更柔和的背景 */
}

QProgressBar::chunk {
    background-color: #0A84FF; /* 更亮的进度颜色 */
    border-radius: 10px; /* 统一圆角 */
}

</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_SentFile">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #0A84FF; /* 按钮颜色 */
    color: white; /* 字体颜色 */
    border: none; /* 移除边框 */
    border-radius: 5px; /* 圆角 */
    padding: 10px 20px; /* 内边距 */
    font: 16px &quot;Helvetica Neue&quot;; /* 字体样式 */
}

QPushButton:hover {
    background-color: #007AFF; /* 悬停时颜色 */
}</string>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
